DESCRIPTION

```diff
@@ -46,6 +46,7 @@ Suggests:
     ggplot2 (>= 3.3.6),
     testthat (>= 3.0.0),
     overlap  (>= 0.3.4),
```

Comment:
Does this need to get removed as well if it's off of CRAN?

Reply:
It doesn't really do anything. Can still download old versions. Not sure what people are going to do.